xain_fl/grpc/cli.py

```diff
@@ -100,10 +100,10 @@ def get_cmd_parameters():
 
     parser.add_argument(
         "-p",
-        dest="num_participants",
+        dest="min_num_participants_in_round",
         default=100,
-        type=type_num_participants,
-        help="Number of participants.",
+        type=type_min_num_participants_in_round,
+        help="Minimum number of participants to be selected for a round.",
     )
 
     parser.add_argument(
```

Comment:
Can we change the default value of `fraction` to `1.0` because if I try to start the coordinator via `python xain_fl/grpc/cli.py  -f file.npy -p 1`, the `minimum_connected_participants` method will return `9.0` which is confusing.

Reply:
good point. i'll update the defaults in `cli.py` to match the new defaults of the `Coordinator`.